texture.js

```diff
@@ -220,9 +220,12 @@ function renderAllTextures(element, game) {
 			for (i = 0; i < textures.length; ++i) {
 				var tx = textures[i];
 				var canvas = tx.drawTexture(ps);
-				//element.appendChild(document.createTextNode(tx.name));
-				element.appendChild(canvas);
-				//element.appendChild(document.createElement("br"));
+                var div = document.createElement("div");
```

Comment:
Formatting looks messed up. Can you switch to tabs?

Reply:
Yeah sorry I forgot about that